gax-java/gax/clirr-ignored-differences.xml

```diff
@@ -13,4 +13,9 @@
     <method>*setWaitTimeout*</method>
     <to>com.google.api.gax.rpc.ServerStreamingCallSettings$Builder</to>
   </difference>
+  <difference>
+    <differenceType>7012</differenceType>
```

Comment:
👍  No rush on reviewing yet! It's still a draft :)

Reply:
Yeah thanks for the reminder! I figured so I didn't review the details, left the comment only in case I forget it next time.

Reply:
I added it here because `@InternalApi` seems to just be an annotation and the clirr test still failed. I can add a comment mentioning this is intended for only google libraries to use

Reply:
I meant that maybe we can do something like 
```
  <difference>
 <className>com/google/api/gax/batching/BatchingDescriptor</className>
  </difference>
```
if possible. If not, we can ignore it in pom, like [this](https://github.com/googleapis/sdk-platform-java/blob/3b85e00357fa24bc0f8c23103cccafdcf9d71b34/gax-java/gax/pom.xml#L90).